test/unit/utilities.js

```diff
@@ -335,10 +335,13 @@ QUnit.test( "Sizzle.uniqueSort", function( assert ) {
 		}
 	};
 
+	// Reset the document after previous tests.
+	Sizzle.setDocument();
```

Comment:
This is not needed on jQuery `main` as there we have a breaking change in `jQuery.uniqueSort` that makes it not depend on the `setDocument()`.

Reply:
Hmm, this may not matter since we invoke `setDocument` inside of `contains` that's used in this API. I'll need to check & remove if not needed anymore.

I'll double check and close the PR if this is not necessary.

Reply:
OK, this is not needed anymore; I removed it from the PR; the PR is now only about not modifying the test input - just like we do it in jQuery tests.